CONTRIBUTING.md

```diff
@@ -36,44 +43,70 @@ The following is a summary of the ideal contribution flow. Please, note that Pul
-- `refactor: ` prefix in the title indicates that PR is only related to refactoring and there is no need to trigger release.
+Pull requests should have a title that follows the specification; otherwise, merging is blocked. If you are not familiar with the specification, simply ask maintainers to modify it. You can also use this cheatsheet if you want:
 
-What about MAJOR release? just add `!` to the prefix, like `fix!: ` or `refactor!: `
+- `fix: ` prefix in the title indicates that PR is a bug fix.
+- `feat: ` prefix in the title indicates that PR is a feature.
+- `docs: ` prefix in the title indicates that PR is only related to the documentation.
+- `chore: ` prefix in the title indicates that PR is only related to cleanup in the project.
+- `test: ` prefix in the title indicates that PR is only related to tests.
+- `refactor: ` prefix in the title indicates that PR is only related to refactoring.
 
-Prefix that follows specification is not enough though. Remember that the title must be clear and descriptive with usage of [imperative mood](https://chris.beams.io/posts/git-commit/#imperative).
+A prefix that follows specification is not enough. Remember that the title must be clear and descriptive with the usage of [imperative mood](https://chris.beams.io/posts/git-commit/#imperative).
 
-Happy contributing :heart:
+Happy contributing! :heart:
 
 ## License
+
 When you submit changes, your submissions are understood to be under the same [Apache 2.0 License](https://github.com/asyncapi/asyncapi/blob/master/LICENSE) that covers the project. Feel free to [contact the maintainers](https://www.asyncapi.com/slack-invite) if that's a concern.
 
-## References
-This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md).
\ No newline at end of file
+## Contribution recognition
+
+We use the [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. For more details read the [`recognize contributors` document](https://github.com/asyncapi/community/blob/master/recognize-contributors.md).
+
+## Maintainers setup
+
+To streamline project management and facilitate onboarding, the maintainer's set up in the project includes two roles: `triager` and `committer`.
+
+There are also two separate areas of responsibility: docs and code. There can be a committer responsible for code only, and there can also be a triager responsible for docs only. There can also be a maintainer that holds all roles and participages in different areas.
+
+We recognize that because of the size of the project and its complexity, areas of responsibility can also get more granular. For example, there can be a committer responsible for docs but only community docs or a committeer responsible only for project design. These exceptions are assessed and approved by project maintainers.
```

Comment:
```suggestion
We recognize that because of the size of the project and its complexity, areas of responsibility can also get more granular. For example, there can be a committer responsible for docs but only community docs or a committer responsible only for project design. These exceptions are assessed and approved by project maintainers.
```
fixed typo